infrastructure/http_metadata_service.go

```diff
@@ -95,6 +95,10 @@ func (ms HTTPMetadataService) GetPublicKey() (string, error) {
 	url := fmt.Sprintf("%s%s", ms.metadataHost, ms.sshKeysPath)
 	resp, err := ms.client.GetCustomized(url, ms.addHeaders())
 	if err != nil {
+		if resp != nil && resp.StatusCode == http.StatusNotFound {
```

Comment:
We refactored `bosh-utils` to no longer return an error on `404`.  Can you update this PR to no longer check the error and just use the status code to determine if you should log this error message?

Reply:
Hi did this update have been released?